gui_qt/user_copy.py

```
@@ -46,6 +46,16 @@
     ),
 }
 
+TASK_PROJECT_GATE_COPY = {
+    "title": "先完成环境检查",
+    "action": "去环境检查",
+    "status_section_title": "任务状态",
+    "project_hint": "请先选择项目",
+    "sync_translation": "选择项目并运行环境检查后，才能开始同步翻译。",
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)